### DISCUSSION TITLE
column formula fields loose formulas on Save

### DISCUSSION THREAD

**Discussion by lukuser** [Budibase/budibase#10857] [Help] [answered]
I notice that recently, when I save a column with a formula, the formulas disappear. I have to hit refresh for them to be accessible again:
![image](https://github.com/Budibase/budibase/assets/81827963/39ae289b-70fa-4bde-9cc5-15471929cc27)
The formula box shows the result of the formula instead of {javascript function} (in this case my formula had an error, so I see the error message). 
On  top of that when I deleted one of the formula-based columns, all the other columns, which are also based on formulas, loose their formulas permanently (they don't return after refresh - I must retrieve them from a backup)

**Comment by melohagan** [comment, marked as answer]
Hey @lukuser 

Thanks for highlighting - there is an open bug issue here: https://github.com/Budibase/budibase/issues/10536
